isos/bootstrap/bootstrap

```diff
@@ -47,6 +47,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
 
     cp /bin/tether ${MOUNTPOINT}/.tether/tether
 
+    echo 'container file system size before copying libraries: '
```

Comment:
Calling this `container file system` is a little misleading - it's the tmpfs specifically for holding the tether, related binaries and files.